manifests/configmap.yaml

```diff
@@ -76,5 +77,5 @@ data:
   # teams_api_url: http://fake-teams-api.default.svc.cluster.local
   # toleration: ""
   # wal_s3_bucket: ""
-  watched_namespace: "*"  # listen to all namespaces
+  watched_namespace: "*" # listen to all namespaces
```

Comment:
IIRC the two spaces are there because of `helm lint`. Please leave it like this.